### DISCUSSION TITLE
valent may take control of defaut call and text app

### DISCUSSION THREAD

**Discussion by alexela2** [andyholmes/valent#351] [General]
I realize that when valent is installed on my pinephone-pro (with phosh via danctnix-arch), it seems like the system has no more default call and text app : for example, when I try to text someone from the contact app, it does not work, neither calling from the chatty… I may also miss many call from the sleep mode due to the same issue. As soon as i uninstall valent, things go well. I use a flatpack version I downloaded on october, 23rd.

I remember having playing around with kde-connect on my original pinephone long ago, and i messed the system up in this way too at that time.

Thanks for your work.

**Comment by andyholmes** [comment]
Oh, dang!

I haven't been using my PinePhone for calls or text so I didn't realize that, sorry! I'm heading to bed, but I can roll out a fix tomorrow.

There's now a [Flatpak nightly](https://valent.andyholmes.ca/valent.flatpakref), so if you install using that it should give you updates with `flatpak update` or GNOME Software.

**Reply by alexela2** [reply]
Dont worry, no emergency. At least, you can run the test to be sure that valent is responsible for this behaviour.
I'll try your «nightly» package (but i only try flatpak ot get valent ;)… arch users are addict to pacman…).

**Reply by andyholmes** [reply]
Should be fixed now hopefully!

I think there's a [package in AUR](https://aur.archlinux.org/packages/valent-git), although I'm not sure if it's kept up to date. Otherwise, there is a [`PKGBUILD`](https://github.com/andyholmes/valent/blob/main/build-aux/misc/PKGBUILD) in the repository I used to use; if you wanted to update that you can (I usually keep the [RPM `valent.spec`](https://github.com/andyholmes/valent/blob/main/build-aux/misc/valent.spec) up to date, which could be used to list the dependencies).

**Reply by alexela2** [reply]
I upgraded. It behaves as expected now. Thanks.
